docs/docs/etl-pipeline-tutorial/4-partition-asset.md

```diff
@@ -14,7 +14,6 @@ sidebar_position: 50
 In this step, you will:
 
 - Create a time-based asset partitioned by month
-- Create a category-based asset partitioned by product category
```

Comment:
Line 7 has a grammatical error -- the comma between "Dagster" and "that allow" should be removed